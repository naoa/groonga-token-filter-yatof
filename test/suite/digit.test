register token_filters/yatof

tokenize TokenBigram \
  "*今日は123雨だ *A!" \
  --normalizer NormalizerAuto \
  --token_filters TokenFilterDigit

