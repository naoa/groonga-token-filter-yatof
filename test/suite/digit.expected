register token_filters/yatof
[[0,0.0,0.0],true]
tokenize TokenBigram   "*今日は123雨だ *A!"   --normalizer NormalizerAuto   --token_filters TokenFilterDigit
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "*",
      "position": 0
    },
    {
      "value": "今日",
      "position": 1
    },
    {
      "value": "日は",
      "position": 2
    },
    {
      "value": "は",
      "position": 3
    },
    {
      "value": "雨だ",
      "position": 4
    },
    {
      "value": "だ",
      "position": 5
    },
    {
      "value": "*",
      "position": 6
    },
    {
      "value": "a",
      "position": 7
    },
    {
      "value": "!",
      "position": 8
    }
  ]
]
